fictionsuit/commands/basics.py

```diff
@@ -18,4 +18,11 @@ async def cmd_summarize(self, message, args):
         `prefix summarize` - returns a summary of the linked article
         """
         response = "This command is not yet implemented. Sorry!"
-        await send_long_message(message.channel.send(response))
+        await send_long_message(message.channel, response)
```

Comment:
The message parameter of cmd_ commands is now a UserMessage, and message.reply(response) will handle long or short messages

Reply:
& same comment applies at line 28 & in main.py line 39;

i probably should have removed send_long_message in my last PR